client/src/components/QueryComponent/QueryComponent.js

```diff
@@ -17,6 +19,9 @@ import { ConfirmQueryDialogComponent } from "./ConfirmQueryDialogComponent/Confi
 const defaultQuery = "SELECT * FROM digitaltwins";
 
 const ENTER_KEY_CODE = 13;
+
+const tokenPattern = new RegExp("([a-zA-Z]+)((?:\\.[a-zA-Z]+)*)", "g"); //eslint-disable-line
```

Comment:
Why is eslint disabled for this line?

Reply:
This was something I was testing to enhance the functionality I just forgot to Remove it